### PR TITLE
Update xcbeautify to 0.13.0

### PR DESCRIPTION
**Pull request by regularberry**
### Short description 📝

XCBeautify has a new 0.13.0 release that fixes a sporadic hang: https://github.com/tuist/xcbeautify/releases/tag/0.13.0

When I tried to run `./fourier update xcbeautify` I found that `build_fat_release_binary` required an `xcode_paths` parameter now. So I added that to the `update` `xcbeautify` and `swiftformat` commands.

### How to test the changes locally 🧐

`./fourier update xcbeautify` and `./fourier update swiftformat` should work now

### Checklist ✅

- [x] The code architecture and patterns are consistent with the rest of the codebase.
- [x] The changes have been tested following the [documented guidelines](https://docs.tuist.io/contributors/testing-strategy/).
- [ ] The PR includes the label `changelog:added`, `changelog:fixed`, or `changelog:changed`, whenever it should be included in the “Added”, “Fixed” or “Changed” section of the CHANGELOG. Note: when included in the CHANGELOG, the title of the PR will be used as entry, please make sure it is clear and suitable.
- [x] In case the PR introduces changes that affect users, the documentation has been updated.
- [x] In case the PR introduces changes that affect how the cache artifact is generated starting from the `TuistGraph.Target`, the `Constants.cacheVersion` has been updated.

### CODE REVIEW

**Comment by danyf90** [comment]
The Xcode version should be saved in a local .xcode-version file, have a look at the release script for reference

**Review by danyf90 — CHANGES_REQUESTED** [review]

File: projects/fourier/lib/fourier/commands/update.rb [hunk 1]
```diff
@@ -18,10 +49,40 @@ def swiftlint
       end
 
       desc "xcbeautify", "Update the vendored xcbeautify binary"
+      option(
+        :xcode_version,
+        desc: %(
+          The version of Xcode to use to build tuist.
+          Can be either a version number (e.g. `--xcode_version="13.2.1"`)
```
Comment (fortmarek) [inline projects/fourier/lib/fourier/commands/update.rb:56]: As Danielle has mentioned, let's use the version in the `.xcode-version` file

**Review by fortmarek — COMMENTED** [review]
After the sourcing of Xcode version, I think we can merge this. Thanks for fixing this!

File: projects/fourier/lib/fourier/commands/update.rb (hunk 1 above)
Comment (regularberry) [reply projects/fourier/lib/fourier/commands/update.rb:56]: Sorry I don't quite understand. Should I remove these parameters? How do you want me to use the `.xcode-version` file?

Comment (danyf90) [reply projects/fourier/lib/fourier/commands/update.rb:56]: Yep 👍

**Comment by fortmarek** [comment]
Hey @regularberry 👋 

let's try to rebase and see if the tests are resolved or not.

**Review by danyf90 — APPROVED** [review]

**Comment by regularberry** [comment]
Dang, did I botch this?

**Pull request closed without merging**